src/model/Account.js

```diff
@@ -51,7 +51,7 @@ export class Account {
   }
 
   setName(name) {
-    if (name == '' || name == undefined) {
+    if (name == '' || name == ' ' || name == undefined) {

```

Comment:
aqui estamos considerando o caso do nome ser `" "`. mas e se for `"  "`? 😅 . o ideal seria considerar name.trim(), algo nesse sentido:

```js
  if (name && name.trim() !=='') 
```